server.js

```diff
@@ -25,6 +27,8 @@ const config = require('nconf')
 	.file('environment', { file: `config/${process.env.NODE_ENV}.json` })
 	.file('defaults', { file: 'config/default.json' });
 
+if(!process.env.NODE_ENV) {process.env.NODE_ENV = (config.get('environment') || 'local');}
```

Comment:
If you want to set the default environment to 'local' (which is fine), it makes more sense to me to move this up before the configs are loaded up, and not load the environment from the config files. i.e., just:

```
if(!process.env.NODE_ENV)
    process.env.NODE_ENV = 'local';
```

Otherwise, we are loading the environment from the config files themselves, which could lead to some weird circular logic (Load up config file depending on environment -> use config file to set environment -> load up config file depending on environment).

If the environment needs to be set automatically, I'm thinking it should be done through the startup script, rather than based on a config file.

Reply:
I've removed the NODE_ENV line from `server.js` and the `environment` variable from the `config/default.json`. I've updated the service file to include them in the "Env Setup" section, which appears to be working correctly.